assignment/api/src/java/org/sakaiproject/assignment/api/model/Assignment.java

```diff
@@ -179,8 +179,7 @@
     @Cache(usage = CacheConcurrencyStrategy.NONSTRICT_READ_WRITE)
     @ElementCollection
     @MapKeyColumn(name = "NAME")
-    @Lob
-    @Column(name = "VALUE", length = 65535)
+    @Column(name = "VALUE", length = 4096)
```

Comment:
Max size for this datatype in Oracle is 4000.

Reply:
Thank you Brian, updated